Pylint: W0107

```bash
mmpose/models/detectors/base.py:32:8: W0107: Unnecessary pass statement (unnecessary-pass)
mmpose/models/detectors/base.py:37:8: W0107: Unnecessary pass statement (unnecessary-pass)
mmpose/models/detectors/base.py:42:8: W0107: Unnecessary pass statement (unnecessary-pass)
mmpose/models/detectors/base.py:47:8: W0107: Unnecessary pass statement (unnecessary-pass)
mmpose/models/backbones/cpm.py:147:8: W0107: Unnecessary pass statement (unnecessary-pass)
mmpose/models/backbones/hourglass.py:185:8: W0107: Unnecessary pass statement (unnecessary-pass)
mmpose/models/backbones/base_backbone.py:46:8: W0107: Unnecessary pass statement (unnecessary-pass)
```

--- a/mmpose/models/backbones/base_backbone.py
+++ b/mmpose/models/backbones/base_backbone.py
@@ -40,4 +40,3 @@ class BaseBackbone(nn.Module, metaclass=ABCMeta):
             x (tensor | tuple[tensor]): x could be a Torch.tensor or a tuple of
                 Torch.tensor, containing input data for forward computation.
         """
-        pass

--- a/mmpose/models/backbones/cpm.py
+++ b/mmpose/models/backbones/cpm.py
@@ -144,7 +144,6 @@ class CPM(BaseBackbone):
                     constant_init(m, 1)
         else:
             raise TypeError('pretrained must be a str or None')
-        pass
 
     def forward(self, x):
         """Model forward function."""

--- a/mmpose/models/backbones/hourglass.py
+++ b/mmpose/models/backbones/hourglass.py
@@ -182,7 +182,6 @@ class HourglassNet(BaseBackbone):
                     constant_init(m, 1)
         else:
             raise TypeError('pretrained must be a str or None')
-        pass
 
     def forward(self, x):
         """Model forward function."""

--- a/mmpose/models/detectors/base.py
+++ b/mmpose/models/detectors/base.py
@@ -26,22 +26,18 @@ class BasePose(nn.Module):
     @abstractmethod
     def forward_train(self, img, img_metas, **kwargs):
         """Defines the computation performed at training."""
-        pass
 
     @abstractmethod
     def forward_test(self, img, img_metas, **kwargs):
         """Defines the computation performed at testing."""
-        pass
 
     @abstractmethod
     def simple_test(self, img, img_metas, **kwargs):
         """Simple test function."""
-        pass
 
     @abstractmethod
     def forward(self, img, img_metas, return_loss=True, **kwargs):
         """Forward function."""
-        pass
 
     @staticmethod
     def _parse_losses(losses):

